panel/vite.config.js

```diff
@@ -1,29 +1,28 @@
+import fs from "fs";
+import path from "path";
+import { defineConfig } from "vite";
 import { createVuePlugin } from "vite-plugin-vue2";
 import pluginRewriteAll from "vite-plugin-rewrite-all";
 import postcssCsso from "postcss-csso";
 import postcssLogical from "postcss-logical";
 import postcssDirPseudoClass from "postcss-dir-pseudo-class";
 
-const fs      = require("fs");
-const path    = require("path");
-const process = require("process");
```

Comment:
Isn't this a problem?

Reply:
For consistency, no. Vite will transpile the ESM config to CJS via esbuild before it actually loads it. Thus, let's import it the ESM way just like all the other modules (which also get transpiled to `require` statements).

`process` is like `window` in the browser or `import.meta` in ES modules, thus no importing necessary.

Reply:
👍